lib/Driver/ToolChains/Flang.cpp

```diff
@@ -286,6 +286,14 @@ void FlangFrontend::ConstructJob(Compilation &C, const JobAction &JA,
     CommonCmdArgs.push_back("8");
   }
 
+  // -Memit-dwarf-common-blocks-name
+  for (auto Arg : Args.filtered(options::OPT_Memit_dwarf_common_blocks_name)) {
+    Arg->claim();
+    CommonCmdArgs.push_back("-x");
```

Comment:
For flang2 only, you need to use `LowerCmdArgs` instead of `CommonCmdArgs`.

Reply:
Yes, this is for flang2 only. And I will make the change.